tests/core/pyspec/eth2spec/test/electra/transition/test_operations.py

```diff
@@ -0,0 +1,98 @@
+from eth2spec.test.context import (
+    ForkMeta,
+    always_bls,
+    with_fork_metas,
+    with_presets,
+)
+from eth2spec.test.helpers.constants import (
+    AFTER_ELECTRA_PRE_POST_FORKS,
+    MINIMAL,
+)
+from eth2spec.test.helpers.fork_transition import (
+    OperationType,
+    run_transition_with_operation,
+)
+
+
+#
+# DepositRequest
+#
+
+@with_fork_metas([ForkMeta(pre_fork_name=pre, post_fork_name=post, fork_epoch=2)
+                  for pre, post in AFTER_ELECTRA_PRE_POST_FORKS])
+@always_bls
+def test_transition_with_deposit_request_right_after_fork(state, fork_epoch, spec, post_spec, pre_tag, post_tag):
```

Comment:
For these three functions, I think we should replace "right" with "immediately" which is more explicit.
```suggestion
def test_transition_with_deposit_request_immediately_after_fork(state, fork_epoch, spec, post_spec, pre_tag, post_tag):
```

Reply:
It’s been copied from the historical operations tests. Do we want to make the name more precise or better to keep it consistent with others? I am fine either way

Reply:
Ah I see. Disregard then. I'll try to fix this in a separate PR later.